### PR TITLE
applications: add MCUboot overlays

### PR DESCRIPTION
**Pull request by jhn-nordic**
Added MCUboot config overlays to asset tracker and
connectivity bridge to ensure backwards compatibility
with bootloaders built pre NCS 1.4

Signed-off-by: Jon Helge Nistad <jon.helge.nistad@nordicsemi.no>

### CODE REVIEW

**Comment by jhn-nordic** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/3214

**Comment by jhn-nordic** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/3214

**Comment by jhn-nordic** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/3212

File: applications/asset_tracker/README.rst [hunk 1]
```diff
@@ -196,6 +196,10 @@ Alternatively, use the command line tool ``menuconfig`` or configure the options
 
 .. external_antenna_note_end
 
+.. note::
+   When ``CONFIG_BOOTLOADER_MCUBOOT=y`` this application is adds an overlay configuration for MCUboot which enables RSA signing of the images for backwards compatbility with MCUboot versions pre NCS 1.4.
```
Comment (jtguggedal) [inline applications/asset_tracker/README.rst:200]: Redundant `is`: `...application is adds...`

Comment (jhn-nordic) [reply applications/asset_tracker/README.rst]: fixed

**Review by jtguggedal — APPROVED** [review]

**Review by nordic-auko — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by nordic-auko — APPROVED** [review]

**Review by jtguggedal — APPROVED** [review]
LGTM

**Comment by greg-fer** [comment]
@shantha-14 , @carlescufi , this is good to go.

**Pull request merged**